File: Java/01 - Introduction to Programming/Task13/names.java
import java.util.*;

public class names
{
    public static void main(String[] args)
    {
        Scanner keyboardInput = new Scanner(System.in);
        //request input from user
        int i = 0;
        String name = "";
        while (!name.equals("stop"))
        {
            System.out.print("Please enter the names of all pupils in class: ");
            name = keyboardInput.nextLine().toLowerCase();
            i+=1;
        }
        
        System.out.println("Number of pupils in class: " + (i-1));

        keyboardInput.close();
    }
}

File: Ruby/01 - Introduction to Programming/Task13/names.rb
#Task13 - names.rb
#------------------------------------------------------------#
#request input from user
i = 0
name = ""
while name!="stop"
    print "Please enter the names of all pupils in class: "
	name = gets.chomp().downcase()
	i+=1
end

puts "Number of pupils in class: " + (i-1).to_s